### ISSUE TITLE
misleading y axis

### ISSUE THREAD

**Issue by lldelisle** [deeptools/pyGenomeTracks#334]
With the follwing ini file:
```
[hlines1]
y_values = 0.1
min_value = 0
max_value = 0.15
file_type = hlines
height = 3

[spacer]

[hlines1]
y_values = 0.01
min_value = 0
max_value = 0.03
file_type = hlines
height = 3

```

We use max_values at 0.15 and 0.03.
In the output:

![h_lines](https://user-images.githubusercontent.com/30404086/113660706-ff7b3680-96a4-11eb-8fd3-aa81ffc7d327.png)

The first y lim is 0.1 instead of 0.15 and the second is 0.0 instead of 0.03.

I really think that keeping artificially only 1 decimal is more painful than useful.

**Comment by rgilsbach** [comment]
We have this issue, too. Would be nice to be able to change digits or units. 
Same problem appears in deeptools profiles and heatmaps.
Thanks for taking care!

**Comment by lldelisle** [comment]
Hi,
I implemented the fix in #343 which adapt the number of decimal or uses scientific notation. It may be part of the next version.

**Comment by rgilsbach** [comment]
Great!

 

Thanks

 

Ralf

 

Von: Lucille Delisle ***@***.***> 
Gesendet: Thursday, June 10, 2021 6:44 PM
An: deeptools/pyGenomeTracks ***@***.***>
Cc: rgilsbach ***@***.***>; Comment ***@***.***>
Betreff: Re: [deeptools/pyGenomeTracks] misleading y axis (#334)

 

Hi,
I implemented the fix in #343 <https://github.com/deeptools/pyGenomeTracks/pull/343>  which adapt the number of decimal or uses scientific notation. It may be part of the next version.

**Comment by lldelisle** [comment]
Solved in version 3.7
